src/Display.jl

```diff
@@ -29,11 +29,12 @@ function git_file_stream(repo::LibGit2.GitRepo, spec::String; fakeit::Bool=false
     return iob
 end
 
-function status(ctx::Context, mode::PackageMode, use_as_api=false)
+function status(ctx::Context, mode::PackageMode, args...; use_as_api=false)
```

Comment:
make `args` the second argument here

Reply:
I don't understand. That's not possible is it?

```
julia> f(x..., y) = @show x
ERROR: syntax: invalid ... on non-final argument
```

Reply:
dont splat it :)